Python/import.c

```diff
@@ -1667,38 +1667,46 @@ PyImport_ImportModuleLevelObject(PyObject *name, PyObject *globals,
         }
     }
     else {
-        static int ximporttime = 0;
+        /* 1 -- true, 0 -- false, -1 -- not initialized */
+        static int ximporttime = -1;
         static int import_level;
         static _PyTime_t accumulated;
         _Py_IDENTIFIER(importtime);
 
         _PyTime_t t1 = 0, accumulated_copy = accumulated;
 
-        Py_XDECREF(mod);
-
         /* XOptions is initialized after first some imports.
          * So we can't have negative cache.
```

Comment:
Is this comment still true? The check for `Py_IsInitialized()` below means that there now *is* a negative cache for `ximporttime == 0`.